### PR TITLE
Add more names from local dump

### PR DESCRIPTION
**Pull request by AaronVanGeffen**

### CODE REVIEW

File: names.idc [hunk 1]
```diff
@@ -423,20 +522,118 @@ static initNames(void)
...
+    MakeName(0x0050CD64, "_treeObjects");
+    MakeName(0x0050CE64, "_snowObjects");
+    MakeName(0x0050CE68, "_climateObjects");
+    MakeName(0x0050CE6C, "_hillShapesObjects");
+    MakeName(0x0050CE70, "_buildingObjects");
+    MakeName(0x0050D070, "_scaffoldingObjects");
+    MakeName(0x0050D074, "_industryObjects");
+    MakeName(0x0050D0B4, "_regionObjects");
+    MakeName(0x0050D0B8, "_competitorsObjects");
+    MakeName(0x0050D138, "_scenarioTextObjects");
```
Comment (AaronVanGeffen) [inline names.idc:588]: Not sure whether these names are already covered by one of the other files — I couldn't find them, at any rate.

Comment (marijnvdwerf) [reply names.idc:588]: These should have been handled by `initObjects`

Comment (AaronVanGeffen) [reply names.idc:588]: I see. I'll remove them, then.

File: structs.idc [hunk 2]
```diff
@@ -140,13 +147,12 @@ static loco_initWindow(void)
     loco_setStructFld(id, 0x040, U16, "number");
     loco_setStructFld(id, 0x042, U32, "flags");
     loco_setStructSub(id, 0x046, "scroll_t", "scroll", 3);
-    loco_setStructVar(id, 0x848, U32);
-    loco_setStructVar(id, 0x84C, U32);
-    loco_setStructVar(id, 0x870, U16);
-    loco_setStructVar(id, 0x872, U16);
+    loco_setStructVar(id, 0x848, "SavedView_t", "saved_view", 1);
```
Comment (AaronVanGeffen) [inline structs.idc:150]: `structs.idc,150: Bad number of arguments for function 'loco_setStructVar'`

**Pull request merged**